wallet-proxy-service/src/lib.rs

```diff
@@ -0,0 +1,6 @@
+/// Configuration of the Wallet Proxy service
+pub mod configuration;
+/// Monitoring API router (health, metrics, etc.)
+mod monitoring_api;
+/// The Wallet Proxy service, including all endpoints
+pub mod service;
```

Comment:
why are some `pub` while others are not? Do we expect to make these available for other workspace members?